File: api/process.js
export default async function handler(req, res) {
  // 设置CORS头
  res.setHeader('Access-Control-Allow-Origin', '*');
  res.setHeader('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE, OPTIONS');
  res.setHeader('Access-Control-Allow-Headers', 'Content-Type, Authorization');

  if (req.method === 'OPTIONS') {
    res.status(200).end();
    return;
  }

  if (req.method !== 'POST') {
    res.status(405).json({ error: '只支持POST请求' });
    return;
  }

  try {
    // 模拟数据处理成功
    res.status(200).json({
      success: true,
      message: '数据处理完成（测试模式）'
    });
  } catch (error) {
    console.error('处理错误:', error);
    res.status(500).json({ 
      error: '处理失败: ' + error.message 
    });
  }
}

File: api/process.py
from flask import Flask, request, jsonify
from datetime import datetime

app = Flask(__name__)

def handler(request):
    """Vercel函数入口点"""
    if request.method == 'OPTIONS':
        return jsonify({'status': 'ok'}), 200, {
            'Access-Control-Allow-Origin': '*',
            'Access-Control-Allow-Headers': 'Content-Type,Authorization',
            'Access-Control-Allow-Methods': 'GET,PUT,POST,DELETE,OPTIONS'
        }
    
    if request.method != 'POST':
        return jsonify({'error': '只支持POST请求'}), 405, {
            'Access-Control-Allow-Origin': '*'
        }
    
    try:
        # 模拟数据处理成功
        response_data = {
            'success': True,
            'message': '数据处理完成（演示模式）',
            'timestamp': datetime.now().isoformat()
        }
        
        return jsonify(response_data), 200, {
            'Access-Control-Allow-Origin': '*'
        }
        
    except Exception as e:
        return jsonify({'error': f'处理失败: {str(e)}'}), 500, {
            'Access-Control-Allow-Origin': '*'
        }